client/src/AppBundle/Resources/views/Admin/Stats/satisfaction.html.twig

```diff
@@ -0,0 +1,39 @@
+{% extends 'AppBundle:Layouts:application.html.twig' %}
+
+{% trans_default_domain "admin" %}
+{% set page = 'satisfactionPage' %}
+
+{% set navSection = 'metrics' %}
+
+{% block htmlTitle %}{{ (page ~ '.htmlTitle') | trans }}{% endblock %}
+{% block pageTitle %}{{ (page ~ '.pageTitle') | trans }}{% endblock %}
+{% block supportTitleTop %}{{ (page ~ '.supportTitle') | trans }}{% endblock %}
+
+{% block helpline %}{% endblock %}
+
+{% block pageContent %}
+    <div class="text">
+        <p>{{ (page ~ '.DescriptionPara') | trans }}</p>
+
+
+        {{ form_start(form, {attr: {class: 'search', novalidate: 'novalidate' }}) }}
+
+        {{ form_known_date(form.startDate, 'satisfactionPage.form.fromDate', {
+            'hintText': 'satisfactionPage.form.fromDate.hint'
+        }) }}
```

Comment:
```suggestion
        {{ form_known_date(form.startDate, 'satisfactionPage.form.fromDate', {
            'hintText': 'satisfactionPage.form.fromDate.hint'
        }) }} <span class="govuk-visually-hidden">of satisfaction search period</span>
```

I'm not sure how this will render but it making sure we're giving context to screen readers wherever we need to. Not crazy about the wording I chose either but you get where I'm going (hopefully)